gulpfile.js

```diff
@@ -6,8 +6,11 @@ const gulp = require('gulp'),
 
 require('dotenv').config({path: 'dot.env'});
 
+const scripts = ['config/*.js', 'data/rooms/*.js', 'data/*.js', 'lib/**/*.js',
+                  'test/*.js', 'app.js', 'gulpfile.js'];
+
```

Comment:
It shouldn't. I'm explicitly calling out each subfolder I want to hit rather than using a wildcard in the root, so It's only hitting those folders and files.